lib/bolt/config.rb

```diff
@@ -43,7 +43,7 @@ module Bolt
       },
       pcp: {
         :"task-environment" => 'production',
-        :"local-validation" => true
+        :"local-validation" => false
```

Comment:
Do we need to set this here or can we just treat undef as false in the code?

Reply:
Ah, we can do that.

Reply:
Well, there's a validation check that asserts it's true or false that needs to be updated if we don't default to false.